Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM resin/rpi-raspbian:wheezy-2015-01-15
+FROM resin/rpi-raspbian:jessie-2015-04-22
```

Comment:
@lifeeth should we not just switch it to :jessie tag so it follows the most recent updates?